prisma/migrations/20220427154926_postcomment/migration.sql

```diff
@@ -0,0 +1,27 @@
+/*
+  Warnings:
+
+  - Added the required column `updatedAt` to the `Post` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Post" ADD COLUMN     "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Is there a change to the prisma.schema file that needs to be committed here too?

Reply:
Yes